static_frame/test/unit/test_frame.py

```diff
@@ -6160,9 +6160,32 @@ def test_frame_to_latex_a(self) -> None:
                 \end{table}
                 '''
         self.assertEqualLines(post, msg)
-
-
-
+    
+    def test_frame_to_msgpack_a(self) -> None:
+        records = (
+                (2, 'a', False),
+                (3, 'b', False),
+                )
+        f1 = Frame.from_records(records,
+                columns=('r', 's', 't'),
+                index=('w', 'x'))
+        post = f1.to_msgpack()
+        msg = b'\x84\xa6_index\xc4\x05\x92\xa1w\xa1x\xa8_columns\xc4\x07\x93\xa1r\xa1s\xa1t\xa5_name\xc4\x01\xc0\xa7_blocks\x92\xc4\x05\x93\x02\xa1a\xc2\xc4\x05\x93\x03\xa1b\xc2'
+        self.assertEqual(post, msg)
```

Comment:
Ultimately you will want to test that, given `f2 = Frame.from_msgpack(msg)`, `f1.equals(f2, compare_name=True, compare_dtype=True, compare_class=True)`.

Reply:
thanks! I was having trouble finding an example of another test that worked like this to copy syntax from.

Reply:
tests now using equals method for comparison.